gamedata/tf2.cattr_starterpack.txt

```diff
@@ -721,13 +722,13 @@
 				"signature"			"CTFPlayer::ApplyPushFromDamage()"
 				"linux"
 				{
-					"offset"		"1988"
+					"offset"		"7BAh"
```

Comment:
Should be `78Ch`, I think.  The position you wrote down doesn't line up with the verify bytes.

Reply:
Yep, it's 78Ch. I actually wrote 78Ch in my gamedata file but it looks like I forgot to change it here. Sorry, my bad.